### ISSUE TITLE
Build skipped: license not found

### ISSUE THREAD

**Issue by mbrodala** [Shippable/support#4135]
I just saw a build (2673 of project `58ab0bf60bfe57100058d7a6`) being skipped with the following message:

```
Job skipped
  Info  
License to run this job on node pool "ssdnodes" not found. Please contact Shippable if you think this is a mistake.
```

Since I think this is a mistake, I contact you here. ;-)

We have 4 nodes in the _Success_ state so I wonder what's wrong here suddenly.

**Comment by Kingdutch** [comment]
We are also experiencing this since about 20-30 minutes no _all_ of our builds. This means that Shippable is unusable for us across all our projects at the moment.

**Comment by rageshkrishna** [comment]
@mbrodala @Kingdutch I'd like to personally apologize for this. This happened while I was migrating some of the subscription licenses. A total of 9 subscriptions were affected by this error. The licenses have been corrected now and all builds should be running fine. Please let me know if you are still running in to any issues.

**Comment by mbrodala** [comment]
I can confirm that builds are running fine again now. Can you add a few more details what kind of migration this was and why it was necessary? Is there a way for you to be alerted about issues like these in the future on time before the customers (read: we) notice?

**Comment by rageshkrishna** [comment]
@mbrodala Absolutely, this was a migration that was necessary as we restructured our billing system to support node pools. The migration was being carried out in a phased manner over the last 4 days and had not run in to any issues until today's final batch of subscriptions.

The error was caused because I had accidentally used the wrong license code in one of today's updates.

A total of 150 subscriptions were updated today and, unfortunately, despite my attempt to monitor these subscriptions I did not notice these 9 affected subscriptions in time. I apologize once again that I did not catch it before it came to your attention.
